src/Action/Iterator.php

```diff
@@ -146,11 +146,11 @@ protected function match(array $row, AbstractScope $scope)
         return $match;
     }
 
-    protected function evaluate($v1, $operator, $v2): bool
+    protected function evaluateIf($v1, $operator, $v2): bool
```

Comment:
If is language building block which consists of expression and it consists of operators. So probably `evaluateOperator` :) (which is better than my original proposal:D )

Reply:
evaluateIf is perfect so we keep it.
method evaluates if $v1 = $v2 for instance. Perfectly clear.
No sense in discussing further on such minor points.

Reply:
not true

Reply:
Imagine no knowledge. What says `evaluateIf` to you? and what `evaluateOperator`? With first example, I do not know.

Reply:
It is a protected method and it is quite clear from the signature.

Anything else about the scopes?

Reply:
see https://discord.com/channels/657649422280425523/664534590160896006/722466104529649674

Reply:
I don't see big problem with "evaluateIf", happy to go with @georgehristov as he is the author of the code. not a dealbreaker.

Reply:
My own opinion here is that @georgehristov should have the choice of function name. Options proposed were: "if", "evaluate", "evaluateIf", "checkIf".  I could ask "verifyIf" and "compare" too. Few arguments - "evaluate" imply a deep evaluation of expression, e.g. ability to have nested expressions of sorts. verify() sounds more like a contingency check (like some sort of validator). compare() would imply to have only 2 arguments and return 1,0,-1. 

I do NOT think that "evaluateIf" is miss-leading, but I'm happy if @georgehristov could look at all SUGGESTIONS and decide what he wants to call the function. I will support his decision, (just don't call it "download()") ! :D

Reply:
one option was `evaluateOperator`